### ISSUE TITLE
How can I use 1000G LD reference to perform MTAG based on GWAS from non-EUR population

### ISSUE THREAD

**Issue by jianvhuang** [JonJala/mtag#159]
Hi, I wonder if MTAG use the LD reference file in **ld_ref_panel** folder, even if I do not specify the _--w-ld-chr_ command.
I would like to apply MTAG to GWAS from non-EUR population, how can I prepare the non-EUR 1000G LD reference for my analysis, e.g. EAS, SAS?

Thank you.

**Comment by JonJala** [comment]
Yes, if the panel is not specified, it uses the eur_w_ld_chr included in
MTAG by default.  You should be able to generate an LD score panel using
the ldsc_mod code included with MTAG if you want to make your own.
